tmail-backend/rate-limit/src/main/scala/com/linagora/tmail/rate/limit/plan/api/RateLimitingPlanUserRepository.scala

```diff
@@ -0,0 +1,15 @@
+package com.linagora.tmail.rate.limit.plan.api
+
+import com.linagora.tmail.rate.limit.model.RateLimitingPlanId
+import org.apache.james.core.Username
+import org.reactivestreams.Publisher
+
+trait RateLimitingPlanUserRepository {
+  def create(username: Username, planId: RateLimitingPlanId): Publisher[Unit]
+
+  def delete(username: Username): Publisher[Unit]
+
+  def listUser(planId: RateLimitingPlanId): Publisher[Username]
```

Comment:
What is this needed for? Can we remove this method for now?

Reply:
![image](https://user-images.githubusercontent.com/55171818/153333651-2789f49c-5995-4772-8567-085db72ec536.png)
This should help the admin easier to manage Plan-users mapping?

Reply:
ok